### PR TITLE
NXP-22093: Replaces  paper-input/input type=date with nuxeo-date-picker

### PR DESCRIPTION
**Pull request by pncosta**

### CODE REVIEW

File: elements/nuxeo-admin/nuxeo-search-analytics.html [hunk 1]
```diff
@@ -41,6 +41,10 @@
         width: 125px;
       }
 
+      nuxeo-date-picker {
```
Comment (nelsonsilva) [inline elements/nuxeo-admin/nuxeo-search-analytics.html]: the need for the padding came with the date picker or it was there before (just uglier)?

Comment (pncosta) [inline elements/nuxeo-admin/nuxeo-search-analytics.html]: came with it. on this page we were using regular "input" (without any paper/other styiling)
the 2 paper-inputs together would appear as a siingle straight line.

the  padding i dont that i've added is this one:
https://github.com/nuxeo/nuxeo-web-ui/pull/40/files/11ab50394229db7b2d2148f62734a84e992d57d0#diff-a513a4785f4d1359e59df5c3115c72c2R49

File: elements/nuxeo-admin/nuxeo-workflow-analytics.html [hunk 2]
```diff
@@ -41,8 +41,12 @@
         width: 125px;
       }
 
+      nuxeo-date-picker {
+        padding-left: 8px;
+      }
       paper-dropdown-menu ::content paper-input-container {
         padding: 0;
+        padding-top: 29px
```
Comment (pncosta) [inline elements/nuxeo-admin/nuxeo-workflow-analytics.html]: how to avoid this and still keep the 3 inputs aligned?

Comment (nelsonsilva) [inline elements/nuxeo-admin/nuxeo-workflow-analytics.html]: @adilioaraujo ☝️  ❓

**Comment by pncosta** [comment]
to remember later:
check the tests and see if this breaks any of them due to the change of the input.

**Pull request merged**